Dockerfile

```diff
@@ -1,5 +1,16 @@
 FROM --platform=$BUILDPLATFORM golang:1.20-alpine3.18@sha256:7839c9f01b5502d7cb5198b2c032857023424470b3e31ae46a8261ffca72912a AS builder
 
+# https://github.com/opencontainers/image-spec/blob/main/annotations.md
+LABEL maintainer="TODO"
```

Comment:
Isn't the `maintainer` annotation replaced by `org.opencontainers.image.authors`?

Reply:
Yep, nice catch, `maintaner` is [deprecated](https://docs.docker.com/engine/reference/builder/#maintainer-deprecated):
>[...] To set a label corresponding to the MAINTAINER field you could use:
>
>`LABEL org.opencontainers.image.authors="SvenDowideit@home.org.au"`